src/openrct2/rct2/S6Importer.cpp

```diff
@@ -1349,23 +1349,22 @@ namespace RCT2
                     dst2->SetAnimationIsBackwards(src2->AnimationIsBackwards());
 
                     // Import banner information
-                    dst2->SetBannerIndex(BANNER_INDEX_NULL);
+                    dst2->SetBannerIndex(BannerIndex::GetNull());
                     auto entry = dst2->GetEntry();
                     if (entry != nullptr && entry->scrolling_mode != SCROLLING_MODE_NONE)
                     {
                         auto bannerIndex = src2->GetBannerIndex();
                         if (bannerIndex < std::size(_s6.banners))
                         {
                             auto srcBanner = &_s6.banners[bannerIndex];
-                            auto dstBanner = GetOrCreateBanner(bannerIndex);
+                            auto dstBanner = GetOrCreateBanner(BannerIndex::FromUnderlying(bannerIndex));
                             if (dstBanner == nullptr)
                             {
-                                dst2->SetBannerIndex(BANNER_INDEX_NULL);
+                                dst2->SetBannerIndex(BannerIndex::GetNull());
                             }
                             else
                             {
                                 ImportBanner(dstBanner, srcBanner);
-                                dst2->SetBannerIndex(src2->GetBannerIndex());
```

Comment:
Afaik you shouldn't delete this

Reply:
ImportBanner assigns the id, and CreateBanner assigns the id which was also specified as the argument, I think its safe to say this banner ends up at the specified id.

Reply:
The tests seem to fail specifically on Windows now, I wonder if this is related. However forcing them to have a different id/index than they are actually in seems like a bug to me.

Reply:
I confused this for the banner its self, this has to be this way.